contracts/MassMigrations.sol

```diff
@@ -101,7 +101,7 @@ contract MassMigration is FraudProofHelpers {
         require(
             MerkleTreeUtilsLib.verifyLeaf(
                 stateRoot,
-                RollupUtils.HashFromAccount(fromAccountProof.account),
+                RollupUtilsLib.HashFromAccount(fromAccountProof.account),
```

Comment:
Actually `keccak256(RollupUtilsLib.BytesFromAccount(new_account))` might be slightly preferred here. 
- Pros: HashFromAccount is more readable
- Cons: HashFromAccount has two function jumps. 

So I'm delaying the optimization later.